### DISCUSSION TITLE
Make emote modification combinable

### DISCUSSION THREAD

**Discussion by golgorath** [night/betterttv#6052] [Ideas]
It is currently possible to make an emote wide by putting `w!` in front of the emote string.
Flipping an emote and removing a space in front of it is also possible, but combining these modifiers is not currently possible I think.

It would be nice to be able to stack the modifiers, e.g. `w! h! POGGERS`, which would result in a wide and horizontally mirrored emote.

**Comment by night** [comment]
added with c102c355506daed5b6b988196337158490aa177e
